plugins/janus_nosip.c

```diff
@@ -657,8 +660,15 @@ int janus_nosip_init(janus_callbacks *callback, const char *config_path) {
 				rtp_range_min = rtp_range_max;
 				rtp_range_max = temp_port;
 			}
+			if(rtp_range_min % 2)
+				rtp_range_min++;	/* Pick an even port for RTP */
+			if(rtp_range_min > rtp_range_max) {
+				JANUS_LOG(LOG_VERB, "Wrong port range: %u -- %u\n", rtp_range_min, rtp_range_max);
+				return -1;
```

Comment:
I wouldn't return an error here. I'd swap min and max so that the range is now valid, and turn the `LOG_VERB` to a `LOG_WARN` to clarify the new range we're enforcing.

Reply:
Ok.
This only can happen if min=max=odd_number